spec/index.bs

```diff
@@ -1417,6 +1417,13 @@ applications should generally request the "worst" limits that work for their con
         allowed in {{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
         when creating a {{GPUPipelineLayout}}.
 
+    <tr><td><dfn>maxBindingIndex</dfn>
+        <td>{{GPUSize32}} <td>[=limit class/maximum=] <td>640<sup><a href="https://quoteinvestigator.com/2011/09/08/640k-enough/">*</a></sup>
```

Comment:
I think there's some chance this link could be confusing to some readers, so perhaps a more subtle nod? like...

`Note: 640 "ought to be enough for anybody" who is using the default [=exceeds the binding slot limits|binding slot limits=].`

Reply:
Done! I liked your verbiage on the note so I used it directly.